features/site-overall.feature

```diff
@@ -0,0 +1,14 @@
+Feature: A user visit gigadb website
+  As a gigadb user
+  All front end views display properly
```

Comment:
If we want to be consistent with other feature descriptions then this needs correcting to: `I want to see all front end views displayed properly`